core/templates/pages/exploration-editor-page/translation-tab/services/voiceover-recording.service.ts

```diff
@@ -63,7 +63,7 @@ angular.module('oppia').factory('VoiceoverRecordingService', [
       }
     };
 
-    var _initRecorder = function() {
+    var _initRecorderAsync = async function() {
```

Comment:
Why is this async? It doesn't return a promise.

Reply:
Done! PTAL!